gorush/Dockerfile

```diff
@@ -3,10 +3,10 @@
 # Stage1: build from source
 FROM quay.io/cybozu/golang:1.13-bionic AS build
 
-ARG GORUSH_VERSION="1.11.2"
+ARG GORUSH_VERSION="1.11.2-beta"
 
 WORKDIR /work
-RUN git clone https://github.com/appleboy/gorush.git -b "v${GORUSH_VERSION}" . \
+RUN git clone https://github.com/emiksk/gorush.git -b "v${GORUSH_VERSION}" . \
```

Comment:
Why don't you move it to cybozu-go org?

Reply:
@ymmt2005 

The emiksk/gorush is temporary repository for evaluating that gorush works properly on Neco.
This repo is going to be deleted after the appleboy/gorush supports proxy for APNs, so I think it is enough to create the repo as personal repo.

Should I move this to cybozu-go org?

Reply:
Understood.

Reply:
Thank you for your comment.